### ISSUE TITLE
[Bug]: Long delay when launching the app.

### ISSUE THREAD

**Issue by redscorpse** [FreeTubeApp/FreeTube#5602]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

Long delay when launching the app.

The app (both AppImage and .deb) takes a long time to launch. I have been testing and the last version that does not have this problem is `freetube_0.19.2_amd64.AppImage`.
The current version of flatpak (0.21.3) closes automatically when launched.

### Expected Behavior

I would expect the startup delay to be like v0.19.2 or lower for future versions.

### Issue Labels

accessibility issue

### FreeTube Version

v0.21.3

### Operating System Version

Debian 12

### Installation Method

.AppImage

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.19.2

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by the-burrito-triangle** [comment]
If you are using Gnome, you could try

https://gitlab.gnome.org/3v1n0/gnome-shell-extension-apps-startup-measure

to profile how much longer it takes for the newer versions of FreeTube.

I, myself, am curious how much longer old vs new FreeTube versions take to open.

I noticed recently, that when opened through a terminal, FreeTube often hangs slightly while errors pop up for gl related vsync:

$ ./freetube_0.21.3_amd64.AppImage
[1487612:0902/172119.490523:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!

I see that error every time I open FreeTube via terminal, often up to 3 times ("failed for <n> times!")

**Comment by redscorpse** [comment]
```bash
❯ time ./freetube_0.21.3_amd64.AppImage
real	47.86s
user	3.15s
sys	0.93s
cpu	8%

❯ time ./freetube_0.21.2_amd64.AppImage 
real	47.68s
user	2.56s
sys	0.74s
cpu	6%

❯ time ./freetube_0.21.1_amd64.AppImage 
real	45.47s
user	2.14s
sys	0.58s
cpu	5%

❯ time ./freetube_0.21.0_amd64.AppImage 
real	45.92s
user	1.87s
sys	0.57s
cpu	5%

❯ time ./freetube_0.20.0_amd64.AppImage 
real	45.85s
user	2.17s
sys	0.79s
cpu	6%

❯ time ./freetube_0.19.2_amd64.AppImage           
real	5.97s
user	1.39s
sys	0.49s
cpu	31%

❯ time ./freetube_0.19.1_amd64.AppImage 
real	6.29s
user	3.05s
sys	0.90s
cpu	62%
```

**Comment by redscorpse** [comment]
I think this issue can be closed, the new version does open as fast as v19.
```bash
time ./freetube_0.22.1_amd64.AppImage 
real	6.48s
user	3.15s
sys	0.99s
cpu	63%
```
